Source/Icebreaker/Content/tour.html

```diff
@@ -78,11 +78,11 @@
             </p>
         </div>
         <div class="footer">
-            <button class="primary-button-back" onclick="ShowDivOnClick('introDiv', 'matchesDiv')">
-                Back
+            <button class="primary-button" onclick="CloseWindow()">
```

Comment:
I believe you changed the order of these divs because you used the reverse ordering. Is there a way to right align the divs without needing to change their order?

Reply:
Not that I know of